File: WeatherServer/WeathARServer/src/main/java/de/htw/mm/weathar/model/DayTime.java
package de.htw.mm.weathar.model;

public enum DayTime {

    // 8 o'clock
    MORNING(8),

    // 12 o'clock
    NOON(12),

    //16 o'clock
    EVENING(16),

    // 20 o'clock
    NIGHT(20);

    private int hour;

    DayTime(int hour) {
        this.hour = hour;
    }

    public int getHour() {
        return hour;
    }

}

File: WeathAR/Assets/Scripts/Model/DayTime.cs
using System;

namespace Model
{
    /**
     * Representation of specific times of a day.
     *
     * @author Cong Minh Nguyen, Tuan Tung Tran
     * @date 20.07.2019
     */
    [Serializable]
    public enum DayTime
    {
        // 8 o'clock
        MORNING,

        // 12 o'clock
        NOON,

        //16 o'clock
        EVENING,

        // 20 o'clock
        NIGHT
    }
}